docs/help/support-guidelines.md

```diff
@@ -1,5 +1,86 @@
 # Support guidelines
 
+This page explains the support mechanisms we provide.
+
 !!! warning
 
-    Under Construction
+    Before asking any help, search and read this documentation site carefully.
+    Also, understand and follow our [discussion guidelines](../contributing/discussion-guidelines/index.md) during your discussions.
+
+Choose appropriate resources depending on what kind of help you need:
+
+- Documentation sites
+  - Various information
+- GitHub Discussions
+  - Questions
+  - Unconfirmed bugs
+  - Feature requests
+  - Design discussions
+- GitHub Issues
+  - Confirmed bugs
+- Slack
+  - Instant messaging between contributors
+- ROS Discourse
+  - General topics that should be widely announced
+
+The following sections are detailed descriptions of each resource.
+
+## Documentation sites
+
+[Docs guide](docs-guide.md) shows the list of useful documentation sites.
+Visit them and see if there is any information related to your problem.
+
+Note that the documentation sites aren't always latest and perfect.
+If you find some information is wrong, unclear, or missing in Autoware docs, feel free to submit a pull request following our [contribution guidelines](../contributing/index.md).
+
+!!!warning
+
+    Since this documentation site is still under construction, there are some empty pages.
+
+## GitHub Discussions
+
+If you have a problem with Autoware, check existing issues and questions and search similar issues first.
+
+- [Issues](https://github.com/autowarefoundation/autoware/issues)
+
+  Note that Autoware has multiple repositories listed in [autoware.repos](https://github.com/autowarefoundation/autoware/blob/main/autoware.repos).  
+  It is recommended to search across the repositories.
+
+- [Questions](https://github.com/autowarefoundation/autoware/discussions/categories/q-a)
+
+If no answer was found, create a new question thread [here](https://github.com/autowarefoundation/autoware/discussions/categories/q-a).
+If your question is not answered in a timely manner (about a week), mention maintainers and remind them.
+
+Also, there are other discussion types such as [feature requests](https://github.com/autowarefoundation/autoware/discussions/categories/feature-requests) or [design discussions](https://github.com/autowarefoundation/autoware/discussions/categories/design).
+Feel free to open or join such discussions.
+
+## GitHub Issues
+
+If you have a problem and you have confirmed it is a bug, find the appropriate repository and create a new issue there.
+If you can't determine the appropriate repository, you should not create an issue yet.
+
+!!!warning
+
+    Do not create issues for questions or unconfirmed bugs. If such issues are created, maintainers will transfer them to GitHub Discussions.
+
+If you want to fix the bug by yourself, discuss the approach with maintainers and submit a pull request.
```

Comment:
In this case, do I need to create an Issue? Or can I create a PR directly?

Reply:
It's written here.
https://github.com/autowarefoundation/autoware-documentation/pull/43/files#diff-1aeecae8f389487f8c42feed5ae17f247adf21aea14e59918c3b6ac7f8d1b033R54

You don't have to create an issue for small fixes.

Reply:
However, since this section is for bugs, I guess an issue is already created. :thinking:

Reply:
Thank you. I considered the case that the bug is found and fixed quickly, but this should be a small PR.

Reply:
You can add some examples here.
https://github.com/autowarefoundation/autoware-documentation/pull/43/files#diff-1aeecae8f389487f8c42feed5ae17f247adf21aea14e59918c3b6ac7f8d1b033R58-R61

Reply:
Resolved.